prometheus-metrics-tracer/prometheus-metrics-tracer-otel-agent/pom.xml

```diff
@@ -32,6 +32,10 @@
     </developer>
   </developers>
 
+  <properties>
+    <automatic-module-name>io.prometheus.metrics.tracer.otel.agent</automatic-module-name>
```

Comment:
The Java package name is `io.prometheus.metrics.tracer.otel_agent`.

Reply:
done